test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/AbstractService.java

```diff
@@ -83,15 +56,23 @@
     static final String PRAVEGA_ID = "pravega";
     static final String ZOOKEEPER_OPERATOR_IMAGE = System.getProperty("zookeeperOperatorImage", "pravega/zookeeper-operator:latest");
     static final String IMAGE_PULL_POLICY = System.getProperty("imagePullPolicy", "Always");
+    static final String BOOKKEEPER_ID = "pravega-bk";
+    static final String CUSTOM_RESOURCE_GROUP_BOOKKEEPER = "bookkeeper.pravega.io";
+    static final String CUSTOM_RESOURCE_VERSION_BOOKKEEPER = "v1alpha1";
```

Comment:
you say `alpha` here but you changed it to `beta` above.

Reply:
beta is wrt pravega:static final String CUSTOM_RESOURCE_VERSION_PRAVEGA = "v1beta1";
alpha is for bookkeeper:  static final String CUSTOM_RESOURCE_VERSION_BOOKKEEPER = "v1alpha1"